frontend/src/Components/CreateCommentComponent.tsx

```diff
@@ -257,6 +258,7 @@ export default function CreateCommentComponent(props: CreateCommentProps) {
                 <div className={styles.control}><button disabled={disabledButtons} onClick={() => applyTag('strike')} title="Перечеркнуть" className={styles.strike}>S</button></div>
                 <div className={styles.control}><button disabled={disabledButtons} onClick={() => applyTag('irony')} title="Ирония"><IronyIcon /></button></div>
                 <div className={styles.control}><button disabled={disabledButtons} onClick={() => applyTag('spoiler')} title="Спойлер"><SpoilerIcon /></button></div>
+                <div className={styles.control}><button disabled={disabledButtons} onClick={() => applyTag('expand')} title="Свернуть/Развернуть"><ExpandIcon /></button></div>
```

Comment:
Probably a good idea is to insert an empty `title` attribute (can be done by adding an optional parameter to `applyTag`).